features/support/page_objects/et1/claim_submitted.rb

```diff
@@ -5,51 +5,55 @@ module Et1
       class ClaimSubmitted < BasePage
         include RSpec::Matchers
         # Claim submitted
-        element :page_header, :page_title, 'claim_confirmations.show.header', exact: false
-        section :callout_confirmation, '.govuk-notification-banner' do
-          # Your claim number
-          element :claim_number, '.govuk-notification-banner__title'
-          element :answer, :css, '.number'
-        end
+        element :page_header, :css, 'h1.govuk-panel__title'
         # What happens next
-        element :what_happens_next, :content_header, 'claim_confirmations.show.what_happens_next.header'
-        section :numerical_list, '.govuk-list' do
-          # We'll contact you once we have sent your claim to the respondent and explain what happens next. At present, this is taking us an of average of 25 days.
-          element :send_to_respondent, :paragraph, 'claim_confirmations.show.what_happens_next.send_to_respondent',
-                  exact: false
-          # Once we have sent them your claim, the respondent has 28 days to reply
-          element :next_steps, :paragraph, 'claim_confirmations.show.what_happens_next.next_steps', exact: false
+        element :what_happens_next, :css, 'h2.govuk-heading-m'
+        section :numerical_list, :css, '.govuk-list--bullet' do
+          # We'll check your claim and may contact you if we have any questions
+          element :send_to_respondent_1, :css, 'li:nth-of-type(1)'
+          # We'll contact you once we've sent your claim to the respondents and explain what happens next
+          element :send_to_respondent_2, :css, 'li:nth-of-type(2)'
         end
         # Submission details
         section :submission_details, :css, '.submission-details' do
-          # Download your claim
-          section :download_application, :grid_row_with_col_labelled,
-                  'claim_confirmations.show.download_application.header' do
-            element :download_application_link, :govuk_link, :'claim_confirmations.show.download_application.link_html'
+          # Submission reference
+          section :submission_reference, :css, '.govuk-summary-list__row:nth-of-type(1)' do
+            # Your claim number
+            element :claim_number, :css, '.govuk-summary-list__value'
```

Comment:
This list of items are thought of as questions and answers - hence why this was called answer before - as the name of the piece of information we are looking at is in the first column - the question.  So in this case, submission_reference.  So, submission_refererence.claim_number kind of makes no sense - this is the submission_reference and the data we are extracting is the answer to that 'question'.

Changing this has caused other test failures - there was 1 place that accesses this 'answer' directly rather than via the 'claim number' method that you have updated.